src/org/zaproxy/zap/extension/api/CoreAPI.java

```diff
@@ -1276,6 +1302,25 @@ public void process(HttpMessage msg) {
 		}
 	}
 
+	private static List<Integer> getIds(JSONObject params) throws ApiException {
+		List<Integer> listIds = new ArrayList<>();
+		for (String id : params.getString(PARAM_IDS).split(",")) {
+			String trimmedId = id.trim();
```

Comment:
It's a pretty minor detail, it can be left as is. 

IMHO streams are less readable, maybe because I'm just not used to them yet...